README.md

```diff
@@ -2493,3 +2493,4 @@ Gaurav Ratnawat ([@gauravratnawat](https://github.com/gauravratnawat)), `Fullsta
 - Rizky Tegar Pratama ([@rizkytegar](https://github.com/rizkytegar)), Full stack web developer, Indonesia
 - Bhavesh Mandhan ([@bhavesh0327](https://github.com/bhavesh0327)), Undergrad at NIT Trichy, India
 - Premkumar ([@PremApk](https://github.com/PremApk)), Software Engineer at Temenos, TamilNadu, India. ❤
+- Eyuel ([@Eyuel](https://github.com/elysium09)), Android Developer, Addis Ababa, Ethiopia. ❤️
```

Comment:
Hey Eyuel, You should place your username on the (`@Eyuel`) part

Reply:
will do

Reply:
thanks @brookmg